doc: add info on fota in bt mesh to nRF52/53 ug (#8912)
nrfconnect/sdk-nrf: doc/nrf/ug_nrf52_developing.rst

    west flash

Make sure to indicate the :file:`overlay-bt.conf` overlay configuration for the Bluetooth transport like in the command example.
This configuration was carefully selected to achieve the maximum possible throughput of the FOTA update transport over Bluetooth with the help of the following features:

* Bluetooth MTU - To increase the packet size of a single Bluetooth packet transmitted over the air (:kconfig:option:`CONFIG_BT_BUF_ACL_RX_SIZE` and others).
* Bluetooth connection parameters - To adaptively change the connection interval and latency on the detection of the SMP service activity (:kconfig:option:`CONFIG_MCUMGR_SMP_BT_CONN_PARAM_CONTROL`).
* MCUmgr packet reassembly - To allow exchange of large SMP packets (:kconfig:option:`CONFIG_MCUMGR_SMP_REASSEMBLY_BT`, :kconfig:option:`CONFIG_MCUMGR_BUF_SIZE` and others).

Consider using these features in your project to speed up the FOTA update process.

.. fota_upgrades_end

.. _ug_nrf52_developing_fota_in_mesh:
.. fota_upgrades_bt_mesh_start

FOTA in Bluetooth mesh
======================

> To perform FOTA update when working with the Bluetooth mesh protocol, use point-to-point Device Firmware Update (DFU) over Bluetooth Low Energy as described in `FOTA over Bluetooth Low Energy`_ above.

Reviewer: ```suggestion
To perform FOTA update when working with the Bluetooth® mesh protocol, use point-to-point Device Firmware Update (DFU) over Bluetooth® Low Energy as described in `FOTA over Bluetooth Low Energy`_ above.
```

> See also :ref:`ug_bt_mesh_fota` for more details about FOTA in Bluetooth mesh.

Reviewer: ```suggestion
See also :ref:`ug_bt_mesh_fota` for more details about FOTA in Bluetooth® mesh.
```

.. fota_upgrades_end

.. ug_nrf52_developing_fota_in_mesh:
.. fota_upgrades_bt_mesh_start

FOTA in Bluetooth mesh
======================

In |NCS|, there is currently no support for Device Firmware Update (DFU) over Bluetooth mesh when performing FOTA.

> However, support for point-to-point DFU using MCUmgr component is available.

Reviewer 2: Should mention BLE here?
```suggestion
However, support for point-to-point DFU over Bluetooth Low Energy using MCUmgr component is available.
```

To perform FOTA upgrade when working with the Bluetooth mesh protocol, use DFU over Bluetooth Low Energy on either smartphone or PC command line tool, following the instructions described in `FOTA over Bluetooth Low Energy`_.

.. note::
      When performing FOTA upgrade on a Bluetooth mesh device and the device's composition data is going to change after the upgrade, unprovision the device before downloading the new image.

See `FOTA upgrade sample`_ for details on how to use the :ref:`zephyr:smp_svr_sample` to set up your project to support FOTA upgrades in |NCS|.

.. fota_upgrades_bt_mesh_end

> The :ref:`bluetooth_mesh_light` sample provides support for point-to-point DFU over SMP for supported nRF52 Series development kits, using a configuration overlay file :file:`overlay-dfu.conf`.

Reviewer 2: I think it is better to be consistent with previous paragraphs:
```suggestion
The :ref:`bluetooth_mesh_light` sample provides support for point-to-point DFU over Bluetooth Low Energy for supported nRF52 Series development kits, using a configuration overlay file :file:`overlay-dfu.conf`.
```

.. fota_upgrades_end

.. ug_nrf52_developing_fota_in_mesh:
.. fota_upgrades_bt_mesh_start

FOTA in Bluetooth mesh
======================

In |NCS|, there is currently no support for Device Firmware Update (DFU) over Bluetooth mesh when performing FOTA.
However, support for point-to-point DFU over Bluetooth Low Energy using MCUmgr component is available.

> To perform FOTA upgrade when working with the Bluetooth mesh protocol, use DFU over Bluetooth LE on either smartphone or PC command line tool, following the instructions described in `FOTA over Bluetooth Low Energy`_ above.

Reviewer: Bluetooth LE or Bluetooth Low Energy, mix in these files.
Is the "use DFU over Bluetooth LE on either smartphone or PC command line tool" necessary? Is that not described in 'FOTA over Bluetooth Low Energy'

.. note::
      If you are using the `nRF Connect Device Manager`_ mobile app to perform FOTA upgrades described in `FOTA over Bluetooth Low Energy`_, the Bluetooth mesh device node might not be visible in the list of devices.
	  See :ref:`ug_bt_mesh_discoverable_dm_app` for more information and steps to be taken before continuing the FOTA upgrade.

.. note::

>       When performing FOTA upgrade on a Bluetooth mesh device and the device's composition data is going to change after the upgrade, unprovision the device before downloading the new image.

Author: I do not disagree with you. Struggling to find a good solution to this issue...

.. fota_upgrades_end

.. fota_upgrades_bt_mesh_start

> FOTA in Bluetooth mesh

Reviewer: ```suggestion
FOTA in Bluetooth® mesh
```
I am not certain if the copy right should be added to Bluetooth here and other places.
